src/groupeddataframe/grouping.jl

```diff
@@ -234,9 +326,9 @@ combine(d -> sum(skipmissing(d[:c])), gd)
 of `combine(map(f, groupby(df, cols)))`.
 
 """
-function combine(f::Function, gd::GroupedDataFrame)
+function combine(f::Any, gd::GroupedDataFrame)
     if length(gd) > 0
-        idx, valscat = _combine(wrap(f(gd[1])), f, gd)
+        idx, valscat = _combine(f, gd)
```

Comment:
Actually you do `makeunique=true` here - at least in `combine`.

Reply:
Yeah, that's https://github.com/JuliaData/DataFrames.jl/issues/1460.